neurovault/apps/statmaps/templates/statmaps/search.html

```diff
@@ -67,11 +67,15 @@
 		    var s = document.getElementsByTagName('script')[0];
 		    s.parentNode.insertBefore(gcse, s);
 		  })();
+
 		</script>
-		<gcse:search linkTarget="_self"></gcse:search>
+		<!-- <gcse:search linkTarget="_self"></gcse:search> -->
+    <gcse:searchbox-only resultsUrl="http://neurovault.org/search"></gcse:searchbox-only>
```

Comment:
if you use:

    <gcse:searchbox-only resultsUrl="search"></gcse:searchbox-only>

it will work in both production and development (this is what I meant by a relative URL).